app/models/metric_rollup.rb

```diff
@@ -47,4 +47,12 @@ def extract_from_min_max(col)
       return val
     end
   end
+
+  def self.get_latest_hourly_rollups(resource_type, resource_ids)
+    metrics = where(:resource_type => resource_type, :capture_interval_name => "hourly")
+    metrics = metrics.where(:resource_id => resource_ids) if resource_ids
+    metrics = metrics.order(:resource_id, :timestamp => :desc)
+    metrics = metrics.select('DISTINCT ON (metric_rollups.resource_id) metric_rollups.*')
+    metrics.includes(:resource)
+  end
```

Comment:
methods in ruby typically don't start with "get", it's more of a Java convention :)
also, indeed why such a very specific method for hourly is needed here? ( as opposed to other intervals)